src/openms/include/OpenMS/METADATA/PeptideIdentification.h

```diff
@@ -166,6 +167,28 @@ namespace OpenMS
     /// returns all peptide hits which reference to a given protein accession (i.e. filter by protein accession)
     static std::vector<PeptideHit> getReferencingHits(const std::vector<PeptideHit>&, const std::set<String>& accession);
 
+      /**
+      @brief Builds MultiMap over UID and pair of index of ConsensusFeature in ConsensusMap and index of PeptideIdentification
```

Comment:
`..b over UID (as obtained from buildUIDFromPepID()) ...`

Reply:
` ... index of PI therein, i.e. cm[p.first].getPeptideIdentifications()[p.second];`